### ISSUE TITLE
Building into one file with grunt still triggers worker-* file loading

### ISSUE THREAD

**Issue by tot-ra** [ajaxorg/ace#2413]
I have a local build where I concat all required files into one file, including ace, but when I start initialization, I still get network request for worker-javascript.js. Is there a one-file build possibility?

Stack trace:
```u
createWorker
$startWorker
$onChangeMode
t.loadModule
setMode
```

**Comment by nightwing** [comment]
worker-javascript.js is loaded into web worker, so it have to be in a separate file.

**Comment by tot-ra** [comment]
Maybe this will help
http://stackoverflow.com/questions/5408406/web-workers-without-a-separate-javascript-file

**Comment by nightwing** [comment]
Yes i am aware that putting whole worker into huge string can help, but that doesn't seem like a good idea. Because it will slow down the time for showing main interface, in order to slightly speed up loading of the syntax checker, which is needed only much later.
